bokeh/document/document.py

```diff
@@ -861,6 +861,8 @@ def set_select(self, selector, updates):
             None
 
         '''
+        if (issubclass(selector, Document)):
```

Comment:
This should also check for `Model` subcalsses, not `Document`. But also I forget this will need to be a bit more careful, and make sure that the selector is a class first before trying to call `issubclass`:
```
from inspect import isclass

if isclass(selector) and issubclass(selector, Model)
```

Reply:
I actually did it :+1: That is why I originally faced import error.